### PR TITLE
Feat/187: Embed fonts in application and fix/185: Get project ubuntu ready 

### LINKED ISSUE #185: Get project ubuntu ready
There are several problems right now:

- pipenv isnt installing 
    - At least the 3.7 python version thats added in the pipenv file is a problem
    - pip needs to be installed separately
    - Even after fixing the 3.7 version pipenv doesnt install anything 
- python is named python3 -> npm run flask not working
- pip is named pip3 -> Server is not starting up, he's trying to install the packages via pip

It is possible to run the application when installing all needed packages manually, but the pip problem will kill all ready built runs, since we cannot install needed packages on startup

---

notes for the documentation:

- nodejs and npm are separate packages

### LINKED ISSUE #187: Embed fonts in application
We still load them from google, need to get rid of that.

### PR DESCRIPTION
**Pull request by diegopf**
First PR! :) 
Closes #187 and fixes #185

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,6 @@
     <base href="./" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="assets/logo.png" />
-    <!-- TODO serve them out of the app, so that its completely offline ready -->
-    <link href="https://fonts.googleapis.com/css?family=Noto+Serif|Roboto:300&display=swap" rel="stylesheet" />
```
Comment (s-blu) [inline src/index.html:10]: Yay, I can complain at other peoples work now _also_ on my free time!!
We'd not only need Roboto, but also Noto Serif, which is used as the font for the editor. :)

File: server/initialize_env.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 from pathlib import Path
 
 from writerey_config import basePath
-
+# TODO: is this package named pip under ubuntu, too?
```
Comment (s-blu) [inline server/initialize_env.py:12]: Oopsie, my bad. Could you remove that for me?

**Review by s-blu — CHANGES_REQUESTED** [review]
The app is now using Roboto in different font weights - depending on the material css - and not everywhere in 300 anymore, like before, simply because it was the only weight available. 

Not sure how we could fix that. Maybe embeding only the 300 weight as the font file? Since material is overwriting that everywhere it seems like a hassle to solve that css wise

**Comment by diegopf** [comment]
@s-blu not sure why but I don't have the problem with Roboto on my machine. I didn't see a difference between development and this branch. Anyway, I've adjusted the configuration to load Roboto and now everything should look the same (I hope! :D)

**Review by s-blu — APPROVED** [review]

**Pull request merged**